airflow/bin/cli.py

```diff
@@ -324,7 +324,12 @@ def task_state(args):
 
 def list_dags(args):
     dagbag = DagBag(process_subdir(args.subdir))
+    print("-" * 60)
```

Comment:
Nit: Might be a bit cleaner to have one print with .format and explicitly include the dashes (textwrap.dedent is a nice way to do this without needing quotes on every line)